core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -422,7 +403,30 @@ public static boolean useEpoll() {
      * {@code -Dcom.linecorp.armeria.useOpenSsl=false} JVM option to disable it.
      */
     public static boolean useOpenSsl() {
-        return USE_OPENSSL;
```

Comment:
I think we still need the null check that returns the value after we set it once like we had before (can use a different name for the local variable if it makes it confusing).